src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    abstract void draw();
```

Comment:
```suggestion
     void draw();
```

abstract is by default in interfaces